emails/2023-08-25.md

```diff
@@ -0,0 +1,29 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+Just a reminder this week [that Fireship exists and he's funnier and more daring than me](https://www.youtube.com/watch?v=jb2AvF8XzII&t=128s).
+
+### Pull Requests in Your App
+
+The core Dolt early adopter [use cases](https://docs.dolthub.com/introduction/use-cases) are starting to form. People tend to use Dolt for one of three things:
+
+1. Dolt as a feature store for an ML pipeline.
+2. Dolt as the database for an app that needs pull requests.
```

Comment:
Put this first?

Reply:
Need to update number below if so

Reply:
I kinda went by popularity and feature store is definitely tops.